Foundation.hs

```diff
@@ -134,6 +134,9 @@ embedLayout widget = do
     pc <- widgetToPageContent widget
     withUrlRenderer $(hamletFile "templates/embed-layout-wrapper.hamlet")
 
+analytics :: App -> Maybe (Entity User) -> url -> Html
```

Comment:
It's a type variable. I figured I could give it *some* information, vs an opaque `a` or `t`. The argument goes unused because we don't do any URL interpolation in this template.